packages/typescript/devserver/BUILD.bazel

```diff
@@ -55,3 +47,11 @@ filegroup(
     tags = ["manual"],
     visibility = ["//visibility:public"],
 )
+
+# BEGIN-DEV-ONLY
```

Comment:
I think we may as well leave these in the distributed file, it means fewer moving parts in deployment of this file, and doesn't cause a transitive dependency to leak

Reply:
👍